test/data/manifests/fedora-coreos-container.mpp.yaml

```diff
@@ -0,0 +1,256 @@
+version: '2'
```

Comment:
Should we create another file (maybe called `fedora-coreos-commit.mpp.yaml`) with an ostree commit as input to the deploy stage instead of having the commit source and input commented out on this file?

Reply:
yeah - definitely some cleanup needed for `fedora-coreos-container.mpp.yaml` - I thought maybe we'd fixup any issues there when we figure out how to properly test this with their test framework (what we discussed earlier today).